mne/viz/evoked.py

```diff
@@ -1024,6 +1024,9 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     if picks is not None:
         pick_names = [evoked.info['ch_names'][pick] for pick in picks]
         evoked.pick_channels(pick_names)
+    else:
+        evoked.pick_types(eeg=True, eog=True, seeg=True)
```

Comment:
Wouldn't I then also have to make `stim=False` explicit?

I think either way it deserves a comment.